content/_index.md

```diff
@@ -91,9 +91,50 @@ To donate conveniently, click on your preferred payment method icon below:
 
 {{ donate_buttons() }}
 
-### Notable sponsors:
```

Comment:
I think this sub-heading was good.
Without it, the following text sticks too close to the row of donations button.